README.md

```diff
@@ -11,7 +11,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [FunctionalConf](https://functionalconf.com/) | 12-16 Dec | The LaLiT Ashok Hotel, Bengaluru | Bring together the growing community of functional programmers under one roof. |
 | [Devops Days India](http://devopsdaysindia.org/) | 8-9 Dec | Sheraton Grand, Bengaluru | Devopsdays is a worldwide series of technical conferences covering topics of software development, IT infrastructure operations, and the intersection between them. | 
 | [PyCon India](https://in.pycon.org/2018/) | 05-09 Oct | Hyderabad International Convention Centre (HICC), Hyderabad | The premier conference in India on using and developing the Python programming language. |
-| [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.|
+| [JSFoo](https://jsfoo.in/2018/) | 26–27 Oct| Bengaluru | The eighth edition of India’s first JavaScript conference.|
```

Comment:
@mukkachaitanya Thanks for fixing this!

Can you place this above PyCon India? The list follows the reverse chronological order of dates.

Reply:
Sorry. Missed doing that. I'll do it ASAP.